src/renderer/views/Channel/Channel.js

```diff
@@ -1991,6 +1991,38 @@ export default defineComponent({
       })
     },
 
+    handleSubscription: function () {
+      // We can't cache the shorts data as YouTube doesn't return published dates on the shorts channel tab
+
+      // Create copies of the arrays so that we only cache the first page
+      // If we use the same array, the store will get angry at us for modifying it outside of the store
+      // when the user clicks load more
+
+      if (this.latestVideos.length > 0 && this.videoSortBy === 'newest') {
+        this.updateSubscriptionVideosCacheByChannel({
+          channelId: this.id,
+          videos: [...this.latestVideos]
+        })
+      }
+
+      if (this.latestLive.length > 0 && this.liveSortBy === 'newest') {
+        this.updateSubscriptionLiveCacheByChannel({
+          channelId: this.id,
+          videos: [...this.latestLive]
+        })
+      }
+
```

Comment:
Should we add a comment about the reasons of not updating shorts cache here?

Reply:
It's already there at the top of the function on line 1995.